shared/constants/types/chat2/meta.tsx

```diff
@@ -19,6 +19,7 @@ export type _ConversationMeta = {
   conversationIDKey: Common.ConversationIDKey // should be the key for this meta EXCEPT for pendingConversationIDKey, in that case its the resolved conversation we're previewing,
   description: string
   descriptionDecorated: string
+  draft: string | null
```

Comment:
The rest of this type uses empty strings in place of `null`. could you do that too?